engine/crates/engine-v2/config/src/v6.rs

```diff
@@ -0,0 +1,123 @@
+use std::{
+    collections::BTreeMap,
+    path::{Path, PathBuf},
+    time::Duration,
+};
+
+use federated_graph::SubgraphId;
+
+use crate::v5::{RateLimitConfigRef, RateLimitRedisConfigRef, RateLimitRedisTlsConfigRef};
+
+pub use super::v5::{
+    AuthConfig, AuthProviderConfig, CacheConfig, CacheConfigTarget, CacheConfigs, EntityCaching, GraphRateLimit,
+    Header, HeaderForward, HeaderInsert, HeaderRemove, HeaderRenameDuplicate, HeaderRule, HeaderRuleId, HeaderValue,
+    JwksConfig, JwtConfig, NameOrPattern, OperationLimits, PathId, RateLimitConfig, RateLimitKey, RateLimitRedisConfig,
+    RateLimitRedisTlsConfig, RateLimitStorage, RetryConfig, StringId, SubgraphConfig,
+};
+
+/// Configuration for a federated graph
+#[derive(serde::Serialize, serde::Deserialize, Debug)]
+pub struct Config {
+    pub graph: String,
```

Comment:
maybe `federated_sdl`? `graph` sounds less helpful

Reply:
I agree. Changing that.